scene/gui/spin_box.cpp

```diff
@@ -69,9 +69,7 @@ void SpinBox::_text_submitted(const String &p_string) {
 	}
 
 	Variant value = expr->execute(Array(), nullptr, false, true);
-	if (value.get_type() != Variant::NIL) {
-		set_value(value);
-	}
+	set_value(value);
```

Comment:
This will happen if the expression is malformed somehow, so I don't think it is correct, but I'm not sure, I'd suggest it should be a separate PR if it is something we want to change

Reply:
I think this change is fine, but it doesn't go well with the added error.

Reply:
Fair enough.